hawkbit-sdk/hawkbit-sdk-device/src/main/java/org/eclipse/hawkbit/sdk/device/DdiTenant.java

```diff
@@ -40,17 +41,26 @@ public void destroy() {
         controllers.clear();
     }
 
-    public DdiController create(final Controller controller, final UpdateHandler updateHandler) {
+    public DdiController createController(final Controller controller, final UpdateHandler updateHandler) {
         final DdiController ddiController = new DdiController(tenant, controller, updateHandler, hawkbitClient);
         controllers.put(controller.getControllerId(), ddiController);
         return ddiController;
     }
 
-    public void remove(final String controllerId) {
-        Optional.ofNullable(controllers.remove(controllerId)).ifPresent(DdiController::stop);
+    public void deleteController(final String controllerId) {
```

Comment:
I'm not sure that deletion from the backend is a part of DdiTenant - this entity represents DDI devices and should not do whatever they don't/can't do

Reply:
my impression is that DdiTenant / DmfTenant are entities that represent Client side logic.
meaning anything you would like to do as a Client (regarding the devices) you can do through these entities.

it is same as removeControlelr (after review unregisterThing on DmfController)

if we it is the case that control of devices (unregister) should be outside these modules, i would remove them

Reply:
agreed, moved mgmt related api calls in new module hawkbit-sdk-mgmt where necessary mgmt calls are placed, hawkbit-sdk-device remains only device related